src/css/Nav.css

```diff
@@ -0,0 +1,47 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  letter-spacing: 0.2px;
+}
+
+.app_container {
+  width: 90%;
+  margin: auto;
+}
+
+.navigation_container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px 0;
+}
+
+.profile_icon {
+  width: 30px;
+}
+
+.nav_tag {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 30px;
+}
+
+.nav_tag h1 {
+  color: #0290ff;
+}
+
+.nav_list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
[OPTIONAL] Your design looks great 👍  But let's DRY your CSS. These 3 line repeating 3 times. Kindly use this as class attributes as mentioned [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👍